src/org/opendatakit/briefcase/export/ExportConfiguration.java

```diff
@@ -154,7 +163,15 @@ public ExportConfiguration setPemFile(Path path) {
   }
 
   public boolean isPemFilePresent() {
-    return pemFile.isPresent();
+    if (pemFile.isPresent())  {
```

Comment:
I think this is going in the right direction, but all these checks should be happening on `setPemFile(Path path)`. The idea is that we don't ever want an `ExportConfiguration` instance to hold an invalid value. We will only set a value on `ExportConfiguration.pemFile` if it's a path to a file with a valid `PrivateKey`

Reply:
This is coming from Value Object design pattern. You can read about it here: https://martinfowler.com/bliki/ValueObject.html